app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if number_of_users_who_made_edits > 1 %}
+              The last user to edit this page was
+            {% endif %}
+              {{ latest_update_events.0.user }}
+            {% if number_of_users_who_made_edits == 1 %}
+              made {{ (all_update_events or (oldest_update_events+latest_update_events))|length }}
```

Comment:
Again, in the case where this falls back to `oldest_update_events+latest_update_events`, this is a value that should be indicated that it's an *at least* value.

Reply:
And also you can't use this value you've called `number_of_users_who_made_edits` as the actual number of users who made edits. You could have an "edit sandwich" and be missing the pages that have edits by a different user.